cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectDockerfile.java

```diff
@@ -95,25 +96,8 @@ private void connectorPluginsPreStage(PrintWriter writer, List<Plugin> plugins)
                     checkForShellMetachars(repo);
                     String artifactHash = Util.sha1Prefix(((MavenArtifact) mvn).getGroup() + ((MavenArtifact) mvn).getArtifact() + ((MavenArtifact) mvn).getVersion());
                     String artifactDir = plugin.getKey() + "/" + artifactHash;
```

Comment:
`plugin.getKey()` is user supplied, right? In which case we need to use `Cmd` on the `mvn dependency:copy-dependencies` part.